internal/version/main.go

```diff
@@ -0,0 +1,51 @@
+package version
+
+import (
+	"fmt"
+	"runtime"
+)
+
+var (
+	version   string
+	buildDate string
+	tempo     string
+)
+
+// Version holds this Operator's version as well as the version of some of the components it uses.
+type Version struct {
+	Operator  string `json:"tempo-operator"`
+	BuildDate string `json:"build-date"`
```

Comment:
Can you update the Makefile and Dockerfile to add the build date and operator version?
The git commit hash would also be helpful imho.

Reply:
This will be added in another PR (it is described in #183). I'm creating small PRs to make them easy to review